spring-native-aws-lambda-function/pom.xml

```diff
@@ -167,14 +167,23 @@
                 <buildArg>--verbose</buildArg>
                 <buildArg>--no-fallback</buildArg>
                 <buildArg>--enable-preview</buildArg>
-                <buildArg>--gc=G1</buildArg>
+                  <buildArg>--gc=G1</buildArg>
                 <buildArg>-march=native</buildArg>
                 <buildArg>--strict-image-heap</buildArg>
                 <buildArg>--enable-url-protocols=http</buildArg>
                 <buildArg>-H:+AddAllCharsets</buildArg>
                 <buildArg>-H:+ReportExceptionStackTraces</buildArg>
               </buildArgs>
             </configuration>
+            <executions>
+              <execution>
+                <id>build-native</id>
```

Comment:
Without this, the `maven-assembly-plugin` fails to include the `spring-native-aws-lambda-function` executable. I think because the compile goal forks the process. So, I make it do it's work in `prepare-package`